mods/beds/functions.lua

```diff
@@ -58,6 +58,20 @@ local function lay_down(player, pos, bed_pos, state, skip)
 		return
 	end
 
+	if bed_pos then
```

Comment:
this only has to happen in the "lay down" case

Reply:
So delete the `if` then?

Reply:
Done

Reply:
?

Reply:
I removed it now.

Reply:
`lay_down` can still be called with `bed_pos == nil` so removing the if here is not correct.

Reply:
here's what I meant by my initial comment: 
![drawing](https://user-images.githubusercontent.com/1042418/73279788-edbf1900-41ed-11ea-9341-e10236d32650.png)

Reply:
Oh ok;
Done!